readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: woocommerce, automattic, woothemes, allendav, kellychoffman, jkudi
 Tags: shipping, stamps, usps, woocommerce, taxes, payment, dhl, labels
 Requires at least: 4.6
 Requires PHP: 5.3
-Tested up to: 5.9
+Tested up to: 6.6.1
```

Comment:
Is this 6.0.2? https://en-ca.wordpress.org/download/releases/

Reply:
Right, this is WP version here. Fixing.